pytorch_lightning/callbacks/lr_monitor.py

```diff
@@ -123,9 +127,17 @@ def _extract_lr(self, trainer, interval: str) -> Dict[str, float]:
                         lr, key = pg['lr'], f'{name}/pg{i + 1}'
                         self.lrs[key].append(lr)
                         latest_stat[key] = lr
+                        if self.log_momentum and pg.get('momentum'):
+                            momentum_key, momentum_value = key + '-momentum', pg.get('momentum')
+                            self.last_momentum_values[momentum_key] = momentum_value
+                            latest_stat[momentum_key] = momentum_value
```

Comment:
keep this in separate method? just to avoid repeated code.

Reply:
does the latest commit look better to you?

Reply:
yep.. better.